curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -505,6 +505,20 @@ impl<C: Command, CE: CommandExecutor<C>> Filter<C, CE> {
         };
         self.update_graph(vid);
     }
+
+    /// Check whether the channel can be shutdown
+    fn check_shutdown(&self) -> bool {
+        self.vs.iter().all(|(_, v)| {
+            matches!(
+                v.inner,
+                VertexInner::Cmd {
```

Comment:
could the vs contains some vertex whose type is not a Cmd, so that it will block the shutdown?

Reply:
Other vs can be executed normally to the end without waiting for a new CEEvent